### PR TITLE
Add SOR-LLC-draft-Ukraine

### PR DESCRIPTION
**Pull request by freevova**

### CODE REVIEW

**Comment by freevova** [comment]
@EnragedWildkin Please review.

File: app/helpers/contracts_helper.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module ContractsHelper
+  def locale_date(date, format, language)
+    I18n.l( date, format: format, locale: language)
```
Comment (EnragedWildkin) [inline app/helpers/contracts_helper.rb]: remove trailing whitespace.

File: app/models/user_field.rb [hunk 2]
```diff
@@ -1,3 +1,15 @@
 class UserField < ActiveRecord::Base
   belongs_to :user, inverse_of: :en_user_field
+
+  def short_name
+    last_name + " " + first_name
```
Comment (EnragedWildkin) [inline app/models/user_field.rb]: use `[last_name, first_name].join(' ')`

File: app/models/user_field.rb [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+  end
+
+  def full_name
+    last_name + " " + first_name + " " + middle_name
```
Comment (EnragedWildkin) [inline app/models/user_field.rb]: Same as above

File: app/models/user_field.rb [hunk 4]
```diff
@@ -1,3 +1,15 @@
...
+  end
+
+  def name_with_initials
+    last_name + " " + first_name[0] + "." + middle_name[0] + "."
```
Comment (EnragedWildkin) [inline app/models/user_field.rb]: Same as above

**Comment by EnragedWildkin** [comment]
@freevova Please check my comments.

**Comment by freevova** [comment]
@EnragedWildkin Please review.

**Comment by EnragedWildkin** [comment]
@freevova Please rebase

**Pull request closed without merging**